### PR TITLE
!audioset status

### PR DESCRIPTION
**Pull request by irdumbs**
git git git git git
trying to rebase
just gonna make a new one.
#296

### CODE REVIEW

**Comment by irdumbs** [comment]
moved checklist here.
clarification from discussion. changes:
- [x] _ for helpers
- [x] helpers up top
- [x] alphabetized
- [x] create_task inside _stop instead of asyncing it
- [x] more functions
- [x] logging
- [x] disable means don't mess with it
- [x] test it

File: cogs/audio.py [hunk 1]
```diff
@@ -906,6 +930,19 @@ def _stop_player(self, server):
             self._kill_player(server)
             del voice_client.audio_player
 
+    # no return. they can check themselves.
+    async def _update_bot_status(self):
+        if self.settings["TITLE_STATUS"]:
+            active_servers = self._get_active_voice_clients()
```
Comment (tekulvw) [inline cogs/audio.py:936]: should really wrap this in a try block

File: cogs/audio.py [hunk 2]
```diff
@@ -1727,6 +1777,8 @@ def has_connect_perm(self, author, server):
                 song = None
             self.queue[server.id]["NOW_PLAYING"] = song
             log.debug("set now_playing for sid {}".format(server.id))
+            await self._update_bot_status()
```
Comment (tekulvw) [inline cogs/audio.py:1780]: I'm gonna suggest `loop.create_task()` here too because this isn't essential to `await`. It'll just slow down the loop

**Comment by irdumbs** [comment]
create_task makes sense.
why try though?
where can it fail?

**Comment by tekulvw** [comment]
if the number of voice clients change while you're iterating over them it'll throw `RuntimeError` iirc

**Comment by irdumbs** [comment]
ty ty
- [x] create_task
- [x] try
- [x] make changes in a new PR cause I can't here anymore :3

**Comment by irdumbs** [comment]
moved to #312

**Pull request closed without merging**